src/Swiper.js

```diff
@@ -612,7 +612,7 @@ export default class SwiperAnimated extends PureComponent {
 
       return (
         <Animated.View
-          key={i}
+          key={"key" in card ? card.key : i}
```

Comment:
Let's use `hasOwnProperty` instead of `in` to ensure stricter matching
 `card.hasOwnProperty('key')`

Reply:
@chitezh done